kanidmd/lib/src/macros.rs

```diff
@@ -736,3 +736,9 @@ macro_rules! vs_cid {
         x
     });
 }
+
+macro_rules! limmediate_warning {
```

Comment:
l'immediate? Je ne comprends pas le Français?

Reply:
Hah - it was an old leftover from log_immediate. It can be removed soon because tracing-forest has a handler for this. I guess even a 4 year old crab can have tech debt :)